Userland/Libraries/LibDebug/Dwarf/AddressRanges.cpp

```diff
@@ -57,6 +57,7 @@ void AddressRanges::for_each_range(Function<void(Range)> callback)
         }
```

Comment:
Pretty sure the fallthrough should be above this line, no? The place where it's at right now already has a return, so it isn't even falling through.